### ISSUE TITLE
0.0.3/0.0.4 maven artifacts contain no classes

### ISSUE THREAD

**Issue by keirlawson** [ChristopherDavenport/log4cats#15]
Not sure if something messed up the build process, but it seems that the 0.0.3 and 0.0.4 release jars only have a manifest in them and no classes/bytecode

**Comment by ChristopherDavenport** [comment]
Hmmm, will work to rectify immediately.

**Comment by ChristopherDavenport** [comment]
Are you pulling in log4cats-core/log4cats-log4s build rather than the root that I was using previously? I appear to be incorrectly still be publishing the root which I shall fix, as that root is now just the compile scope for the project, not really containing classes itself.

I think you will likely by fine switching to 
`log4cats-log4s` as I believe you were on log4s. Also the logger switched into a package down with classes directly for log4s.

**Comment by keirlawson** [comment]
Ah yep, I was using the old dependecy.  Getting rid of those empty packages would be good so as to avoid confusion.

**Comment by ChristopherDavenport** [comment]
Yeah, I always wondered why people included those noPublishSettings components, now I know why as I have made the mistake myself. 

Experience being the best teacher and all 😉
